packages/zoe/src/contracts/multipoolAutoswap.js

```diff
@@ -36,13 +36,22 @@ export const makeContract = harden(
     // This contract must have a "central token" issuer in the terms.
     const CENTRAL_TOKEN = 'CentralToken';
 
+    function buildAmountMathKeywordRecord(keywords) {
+      const { issuerKeywordRecord } = zcf.getInstanceRecord();
```

Comment:
Let's create a brandKeyword record accessible here, too, then we don't have to get the brand for each.

Reply:
done